crypto3/libs/blueprint/include/nil/blueprint/zkevm_bbf/opcodes/dummy.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <iostream>
+
+#include <nil/blueprint/zkevm/zkevm_word.hpp>
+#include <nil/blueprint/zkevm_bbf/types/opcode.hpp>
+
+namespace nil {
+    namespace blueprint {
+        namespace bbf{
+            template<typename FieldType>
+            class opcode_abstract;
+
+            // SHOULD NOT BE USED IN THE FINAL VERSION
+            template<typename FieldType>
+            class zkevm_dummy_operation : public opcode_abstract<FieldType> {
+            public:
+                virtual void fill_context(
+                    typename generic_component<FieldType, GenerationStage::ASSIGNMENT>::context_type &context,
+                    const opcode_input_type<FieldType, GenerationStage::ASSIGNMENT> &current_state
+                ) override  {
+                }
+                virtual void fill_context(
+                    typename generic_component<FieldType, GenerationStage::CONSTRAINTS>::context_type &context,
+                    const opcode_input_type<FieldType, GenerationStage::CONSTRAINTS> &current_state
+                ) override  {
+                    std::cout << "DUMMY CONTSTRAINT, REPLACE ME WITH SOME ACTUAL IMPL" << std::endl;
```

Comment:
may be dummy constrain for utilize 1 row as reported in `rows_amount`

Reply:
They are actually needed to skip row from zkevm table without breaking the checks, so seems OK to me

Reply:
ok